prettier.config.js

```diff
@@ -0,0 +1,5 @@
+module.exports = {
```

Comment:
Would it be possible to use [prettier's overrides](https://prettier.io/docs/en/configuration.html#configuration-overrides) to get it to apply to the `.js.es6` files?

Reply:
I think since I modified `package.json` that this isn't necessary.